instagram-app/src/components/PostContainer/useractions/useractions.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './useraction.css';
+import message_logo from './comment-icon.png';
+import like_heart_clear from './Instagram-heart.png';
+import like_heart_red from './Instagram-Heart-liked.png';
```

Comment:
`import like_heart_clear from './Instagram-Heart.png';`

this will match the file name you have in your folder.